packages/elements/src/internals/machines/sign-in/router.types.ts

```diff
@@ -129,5 +118,29 @@ export type SignInRouterSnapshot = MachineSnapshot<
   SignInChildren,
   SignInStateValue,
   SignInRouterTags,
-  SignInOuptut
+  SignInOuptut,
+  any // TMeta - Introduced in XState 5.12.x
+>;
+
+// ---------------------------------- Machine Type ---------------------------------- //
+
+// Used for simple typing of parent refs passed to children
+export type TSignInRouterMachine = StateMachine<
```

Comment:
In this instance the `any`'s are fine. We're only acting on the parent context and events w/in children.